src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -899,10 +899,10 @@ public bool CanWeightComputationBeAvoided(Float instanceDistanceToBestOldCluster
                 else
                 {
                     ArrayDataViewBuilder arrDv = new ArrayDataViewBuilder(host);
-                    arrDv.AddColumn("Features", PrimitiveType.FromKind(DataKind.R4), clusters);
-                    arrDv.AddColumn("Weights", PrimitiveType.FromKind(DataKind.R4), totalWeights);
+                    arrDv.AddColumn(DefaultColumnNames.Features, PrimitiveType.FromKind(DataKind.R4), clusters);
+                    arrDv.AddColumn(DefaultColumnNames.Weight, PrimitiveType.FromKind(DataKind.R4), totalWeights);
                     var subDataViewCursorFactory = new FeatureFloatVectorCursor.Factory(
-                        TrainUtils.CreateExamples(arrDv.GetDataView(), null, "Features", weight: "Weights"), CursOpt.Weight | CursOpt.Features);
+                        new RoleMappedData(arrDv.GetDataView(), null, DefaultColumnNames.Features,weight: DefaultColumnNames.Weight), CursOpt.Weight | CursOpt.Features);
```

Comment:
formatting: space before `weight` parameter.

Reply:
Thanks @zeahmed !

---
In reply to: [200496962](https://github.com/dotnet/machinelearning/pull/496#discussion_r200496962) [](ancestors = 200496962)